src/components/ebay-breadcrumbs/examples/page-custom-attrs.marko

```diff
@@ -1,6 +1,6 @@
 <ebay-breadcrumbs a11yHeadingText="Page navigation">
-    <@item href="http://www.ebay.com/" navsrc="{}" _sp="p2345.m909.l34">eBay</@item>
```

Comment:
I think this is fine but we need to figure out how to pass in "anything" into some components